app/views/comments/_help.html.haml

```diff
@@ -22,6 +23,6 @@
   %p.text-right{:style=>"margin-bottom: 10px;"}
     = link_to "More Complex Markdown Help", "http://daringfireball.net/projects/markdown/syntax", target: "_blank"
 %p.text-right
-  %a{:type=>"button", "data-toggle"=>"collapse", "data-target"=>"#help", :class=>"btn btn-default btn-xs"}
+  %a{:type=>"button", "data-toggle"=>"collapse", "data-target"=>"#help#{unique_id}", :class=>"btn btn-default btn-xs"}
```

Comment:
Please use `:` instead of `=>`

Reply:
done.
Can you please check why my Travis is showing failure in test cases? :/ there's not a single test failure on my local machine.